### PR TITLE
fix: Avoid using boolean types under Oracle's data type limitations #673

### PR DESCRIPTION
**Pull request by Roiocam**
References #673

This PR has been fix something like `EventsByTagTest`, but i got the ERROR on `JournalSequenceActorTest`.

I think it was my environment complaining database too slow, waiting for investigation.

### CODE REVIEW

**Comment by Roiocam** [comment]
@patriknw These CI checks have passed. Does this mean that this PR has successfully solved #673?

File: core/src/main/scala/akka/persistence/jdbc/query/dao/legacy/ByteArrayReadJournalDao.scala [hunk 1]
```diff
@@ -122,7 +122,7 @@ trait OracleReadJournalDao extends ReadJournalDao {
     if (isOracleDriver(profile)) {
       val theOffset = Math.max(0, offset)
       val theTag = s"%$tag%"
-
+      // fixes https://github.com/akka/akka-persistence-jdbc/issues/673, oracle do not support boolean type.
```
Comment (johanandren) [inline core/src/main/scala/akka/persistence/jdbc/query/dao/legacy/ByteArrayReadJournalDao.scala:125]: No need to leave a comment around, the commit history will tell the story
```suggestion
```

**Review by johanandren — COMMENTED** [review]
One small thing then I think this looks good to merge

**Review by johanandren — APPROVED** [review]
LGTM, thanks!

**Pull request merged**